src/unix/freebsd.c

```diff
@@ -58,7 +58,8 @@ void uv__platform_loop_delete(uv_loop_t* loop) {
 
 
 #ifdef __DragonFly__
-int uv_exepath(char* buffer, size_t* size) {
+/* deprecated long ago, use KERN_PROC_PATHNAME sysctl */
+int uv_exepath_procfs(char* buffer, size_t* size) {
```

Comment:
Since this is now unused from anywhere, seems like it should be deleted?

Reply:
Well, I am not the original author of the patch. Why it wasn't simply deleted - I can only guess. To me this indeed should be deleted. I will create a separate PR then.